src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -182,12 +101,13 @@ class PopoverReactionList extends React.Component {
                 emojiCount: 0,
             };
         }
-        const emojiCount = selectedReaction.users.length;
-        const reactionUsers = _.map(selectedReaction.users, (sender) => sender.accountID);
-        const emoji = EmojiUtils.findEmojiByName(selectedReaction.emoji);
+        const reactionUsers = _.pick(selectedReaction.users, _.identity);
+        const emojiCount = _.map(reactionUsers, (user) => user).length;
+        const userAccountIDs = _.map(reactionUsers, (user, accountID) => parseInt(accountID, 10));
```

Comment:
Is there a reason to use `parseInt` instead of `Number()` here?

Reply:
Ok, changed to Number()